children.rst

```diff
@@ -1,8 +1,9 @@
-Children
+Young Hackers & Children
 ========
+Young adults aged between 16 and 18 can join the Hackspace as 'Young Hackers'. The guidance for this is available on request, the key thing to note being that their parents or guardians will need to meet with a Trustee at the Hackspace to discuss the risks within the Space. If you know of someone interested in becoming a Young Hacker, please ask them to contact the `Membership team`__.
 
-The Hackspace welcomes children, but please be aware that it is primarily an adult environment. While of course we won’t be screening porn or having strippers in the space, outside of workshops specifically aimed at kids, members might not be watching their language. Additionally, the space has some very dangerous tools and potentially dangerous chemicals around. Under 18s must  be accompanied by an adult and are generally expected to be engaged with the space in some way, i.e. working on a project of some kind while they’re here.
+Children are welcome at Nottinghack, however be aware that it is primarily an adult environment. While it is unlikely that there would be content overtly unsuitable for children, members may not be watching their language or could be discussing adult topics. In addition to this; the Hackspace has potentially dangerous tools and chemicals which are not locked away, and no specific child-proofing measures in place.
 
-Young adults aged between 16 and 18 can join, but one of their parents or guardians will need to meet with a trustee at the Hackspace to understand the risks involved.
+**Under-18s must be accompanied by an adult**, and are generally expected to be engaged with the space some way (such as working on a project). Periodically we have events specifically for children of all ages.
```

Comment:
Change to:

**Under-16s, and 16-18 year olds who are not 'Young Hackers', must be accompanied by an adult**